hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -638,12 +638,6 @@ possible configurations would overwhelm and obscure the important.
     <value>true</value>
     <description>Whether to merge a region as part of normalization.</description>
   </property>
-  <property>
-    <name>hbase.normalizer.min.region.count</name>
```

Comment:
What's the reasoning for removing the default value from this file? While it's redundant to use in code, this file serves as a source of documentation ; I believe it is parsed by our site building process to populate a section of the online book.

Reply:
If here just change to "hbase.normalizer.merge.min.region.count",  conf.get(MERGE_MIN_REGION_COUNT_KEY) always get value even user not configured, and we can't be backward compatible.

Reply:
I don't understand the backward compatibility concern here, can you explain again for me? The variable `MERGE_MIN_REGION_COUNT_KEY` is not public, just like `MIN_REGION_COUNT_KEY`, so users won't use the variables in their code. The configuration key `"hbase.normalizer.merge.min.region.count"` is new with this patch, so if a user was looking up that value in an existing release, it should error or return no value. We don't need to preserve backward compatibility of a configuration key that is not defined.